grunttasks/serve.js

```diff
@@ -5,9 +5,15 @@
 module.exports = function (grunt) {
   'use strict';
 
-  grunt.registerTask('serve', [
-    'build:development',
-    'hapi',
-    'watch'
-  ]);
+  grunt.registerTask('serve', 'Builds the dist assets, starts the Hapi server, and watches for changes.', function (target) {
+    if (!target) {
+      target = 'development';
+    }
```

Comment:
Still not sure about this `target` stuff. We already have a `NODE_ENV` flag in [server/config.js](https://github.com/mozilla/chronicle/blob/9e7bd617877a9859937772149545e4c61dc6d9dc/server/config.js#L19) which lets us toggle between 'local', 'awsdev', 'prod', 'test'.

Not sure if it is easier to just use `config.get('env')` -- although then we'd have to potentially handle all 4 environments.